apps/labs/posts/pydata-theme-a11y-fixes.mdx

```diff
@@ -0,0 +1,479 @@
+---
+title: 'PyData Theme Accessibility Fixes'
+authors: [gabriel-fouasnon]
+published: June 26, 2024
+description: 'A small showcase of accessibility improvements made to the PyData Sphinx Theme, Fall 2023-Spring 2024'
+category: [Access-centered]
+featuredImage:
+  src: /posts/hello-world-post/featured.png
+  alt: 'Excellent alt-text describing the featured image'
+hero:
+  imageSrc: /posts/hello-world-post/hero.jpeg
+  imageAlt: 'Excellent alt-text describing the hero image'
+---
+
+# PyData Theme Accessibility Fixes, Before and After
+
+Over the past several months, I have been working with a few coworkers and the
+PyData community at large to make some accessibility improvements to the PyData
+Sphinx Theme. For this blog post, I chose three such improvements (out of many!)
+to examine with a before-and-after lens to give a better idea of how these kinds
+of changes can create an improved user experience for disabled people—and often
+for people without disabilities, too.
```

Comment:
This is a small nit-pick, so feel free to ignore if you don't feel the same. As a non-native english speaker, I found the sentence _"For this blog post, I chose...without disabilities, too."_ a bit too long and hard to follow. I think breaking it into two sentences could help.

Something like this maybe:

> For this blog post, I chose three such improvements (out of many!) to examine with a before-and-after lens. This will give a better idea of how these kinds of changes can improve user experience for disabled people—and often
> for people without disabilities, too.